src/service/resources-impl.js

```diff
@@ -859,6 +871,19 @@ export class Resources {
     this.visibilityStateMachine_.setState(
       this.viewer_.getVisibilityState()
     );
+
+    if (this.maybeChangeHeight_) {
+      this.vsync_.measure(() => {
+        const measuredScrollHeight = this.viewport_.getScrollHeight();
+        if (measuredScrollHeight != this.scrollHeight_) {
+          this.viewer_.sendMessage('documentHeight',
+              {height: measuredScrollHeight}, /* cancelUnsent */true);
+          this.scrollHeight_ = measuredScrollHeight;
+          dev().fine(TAG_, 'document height changed: ' + this.scrollHeight_);
+        }
+        this.maybeChangeHeight_ = false;
```

Comment:
Set it to false before `vsync.measure()`. In other words as a first line inside the `if`. This will avoid running several measures unnecessary.